exp/api/v1alpha3/azuremanagedcontrolplane_types.go

```diff
@@ -26,7 +26,6 @@ import (
 type AzureManagedControlPlaneSpec struct {
 	// Version defines the desired Kubernetes version.
 	// +kubebuilder:validation:MinLength:=2
-	// +kubebuilder:validation:Pattern:=^v(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)([-0-9a-zA-Z_\.+]*)?$
```

Comment:
where is prefix normalization happening?

Reply:
https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/6ad50d0e79546d9f3c5e5d632836412df91a6a9c/exp/controllers/azuremangedcontrolplane_reconciler.go#L59

actually should add it to https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/6ad50d0e79546d9f3c5e5d632836412df91a6a9c/exp/controllers/azuremanagedmachinepool_reconciler.go#L61

Reply:
added